src/cosl/coordinated_workers/coordinator.py

```diff
@@ -215,7 +215,8 @@ def __init__(
             nginx_options: Non-default config options for Nginx.
             is_coherent: Custom coherency checker for a minimal deployment.
             is_recommended: Custom coherency checker for a recommended deployment.
-            tracing_receivers: Endpoints to which the workload (and the worker charm) can push traces to.
+            tracing_receivers: A function generating endpoints to which the workload (and the worker charm) can push traces to.
```

Comment:
As we're going to split workload and charm traces, I think this should be for workload traces only. charm traces are always going to be otlp http